src/main.py

```diff
@@ -170,7 +170,12 @@ def utimens(self, path, times=None):
 
     def open(self, path, flags):
         print("open called with path " + path)
-        full_path = path
+        # /home/lon/mdh/test_flat/buildInfo.txt
+        # /fuse_mount/home/lon/mdh/test_flat
+        _, path = path.split("/home/lon/mdh")
+
+        full_path = os.path.join("/app", path)
+        print(full_path)
```

Comment:
please add a comment or something that using this will throw massive amounts of error since this is using your own path ( or remove it)
@ma-schulze